manageiq-gems-pending.gemspec

```diff
@@ -28,6 +28,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "more_core_extensions",    "~> 4.4"
   s.add_runtime_dependency "net-ftp",                 "~> 0.1.2"
   s.add_runtime_dependency "nokogiri",                "~> 1.14", ">= 1.14.3"
+  s.add_runtime_dependency "rexml",                   ">= 3.3.2"
```

Comment:
I'd rather not add it as a dependency, because we want to learn towards Nokogiri anyway, but if you look at the [Gemfile.lock.release](https://github.com/ManageIQ/manageiq/blob/7aa4afff3e9614c919fd03f2f6066fbafe4dfcb9/Gemfile.lock.release#L1147), rexml is already there as a dependency of a number of libraries, including google-apis-core, so we might as well enforce a minimum here.

Reply:
It's complaining about 

```
metadata['rubygems_mfa_required'] must be set to 'true'.
```

that can be done in a followup